Greenstand-Token.md

```diff
@@ -434,9 +430,9 @@ Sealing nodes are compensated using minting and transfer fees based on the value
 
 A TRing is the native currency of the Greenstand network and is created by depositing Greenstand tokens.
 
-The Greenstand Token resides on the network as an ERC721 token. Any Greenstand Token may be converted into TRings as an ERC20. The number of TRrings created from this conversion is calculated using the Greenstand network’s master RVI.
+The Greenstand Token resides on the network as an ERC721 token. Any Greenstand Token may be converted into TRings. The number of TRrings created from this conversion is calculated using the Greenstand network’s master RVI.
 
-The TRing is divisible by 100 units called Grain.
+The TRing is divisible by 100 units called Seed.
```

Comment:
https://academy.binance.com/en/glossary/wei
In my opinion, we should conform to the existing standard to the best of our ability. I will defer to your judgement, however.